doc/articles/concepts/overview/why-uno-platform.md

```diff
@@ -0,0 +1,275 @@
+
+## Keep using your favorite IDE
+
+Need we say more? Use the IDE that works for you – Visual Studio, VS Code, Rider as well as GitHub Codespaces & GitPod. Wherever you are, Uno Platform is with you.
+
+## Work from your favorite OS
+
+Remain on the operating system that you are on. A detailed list of target platforms for which you can develop from Windows, macOS, or Linux is available. Our docs are designed to be OS agnostic, so every tutorial step works regardless of the system you develop on.
+
+## Use either XAML or C# Markup for your app UI
+
+With Uno Platform, you have a choice of building polished cross-platform UI with concise declarative UI markup using a modern XAML-centric syntax or declarative-style C# Markup. Both approaches are supported right out of the box. Our C# Markup approach works with both 1st party and 3rd party controls and application-level libraries, mapping directly to the underlying object model. All thanks to real-time C# source generators doing the heavy lifting for you.
+
+## Hot Reload that just works
+
+Our Hot Reload offers the most comprehensive solution for fast development loop – make a change and see it live everywhere is running in real-time without recompiling. Uno Platform Hot Reload works on:
+
+- Visual Studio and Visual Studio Code
+- All target platforms
+- XAML, C# Markup, and C#
+- Bindings & x:Bind, Resources, Data Templates, and Styles
+- 1st party or 3rd party controls
+- Devices and Emulators
+
+For more information, see [Hot Reload](xref:Uno.Features.HotReload).
+
+## Stop typing markup and use Figma code generation
+
+Design handoff is one of the biggest time traps, as design envisioned by the designer needs to be manually translated to markup. With our plugin\*, this process is automatic, and the result is well structured, performant XAML or C#.
+
+For more information, see [Design and Build Uno Platform Applications with Figma](xref:Uno.Figma.Overview.GetStarted).
+
+\*_Note this plugin is optional_
+
+## Quickly create a project using the Templates Wizard
+
+Our [Uno Platform Template Wizard](https://platform.uno/blog/the-new-uno-platform-solution-template-wizard/) and its [live version](https://new.platform.uno/) allow you to get to a working project quickly. You can select your preferred version of .NET, choose the target platforms you want to develop for, pick a design system that suits your app the best, pick from the MVVM or MVUX architectural pattern, and throw in any mix of the rich set of Uno Extensions as starting building blocks. Additionally, you can include support for PWA, first-party API server, UI tests, and even generate initial build scripts for GitHub Actions or Azure DevOps, alleviating the hassle of figuring out the complex, cross-platform YAML specifics.
```

Comment:
@jeromelaban for links to uno blog, should we be using aka links, or are we ok with these being hard links?

Reply:
Links to the blog are fine, they're less likely to change I think.